authorization-server/jwt-authorization-grant-token-exchange.js

```diff
@@ -29,12 +29,11 @@ export async function authorizationGrantTokenExchange(ctx, configuration, redisC
   }
 
   // Lookup by payload.sub and get the last id assertion saved
-  const { subjectToken } = getSubjectToken(payload.sub, provider.client_id);
-
-  // TODO store in redis for debug console
+  const { subjectToken, tokenType } = getSubjectToken(payload.sub, provider.client_id);
 
   await redisClient.set(`jag_subject_token:${payload.sub}`, subjectToken);
```

Comment:
i forgot to mention in the previous review.  Can you add a comment saying we store this info to redis purely for informational purposes to surface to the debug console?

Reply:
i was referring to a new comment like the one I mentioned in my github comment